packages/contracts/contracts/MarketLiquidityRewards.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+
+/*
+- Claim reward for a loan based on loanId (use a brand new contract)
+- This contract holds the reward tokens in escrow.
+- There will be an allocateReward() function, only called by marketOwner, deposits tokens in escrow
+- There will be a claimReward() function -> reads state of loans , only called by borrower -> withdraws tokens from escrow and makes those loans as having claimed rewards
+- unallocateReward()
+
+This contract could give out 1 OHM when someone takes out a loan (for every 1000 USDC)
+
+
+*/
+ 
+contract MarketLiquidityRewards is 
+InitializableUpgradeable
+{
+    address constant tellerV2;
+    address constant marketRegistry;
+
+
+    //marketId => rewardAllocation
+    mapping(uint256 => RewardAllocation) public allocatedRewards;
```

Comment:
should be able to allocate multiple tokens to a market at once 

anyone can allocate tokens to a market